packages/react-components/src/components/carousel.tsx

```diff
@@ -10,15 +10,18 @@ import Gif, { EventProps, GifOverlayProps } from './gif'
 const carouselCss = css`
     -webkit-overflow-scrolling: touch;
     overflow-x: auto;
+    overflow-y: hidden;
     white-space: nowrap;
     position: relative;
 `
 const carouselItemCss = css`
     position: relative;
     display: inline-block;
     list-style: none;
+    /* make sure gifs are fully visible with a scrollbar */
+    margin-bottom: 1px;
```

Comment:
Instead of changing the threshold, add this one pixel margin to keep the full gif in view even when there are scrollbars